src/TextField/TextFieldLabel.js

```diff
@@ -4,15 +4,14 @@ import React, { PropTypes } from 'react';
 import { createStyleSheet } from 'jss-theme-reactor';
 import classNames from 'classnames';
 import { easing } from '../styles/transitions';
+import LabelBase from '../internal/LabelBase';
```

Comment:
No, I need this `BaseLabel` functionality for a required `RadioGroup` or set of `Checkbox`es.